src/worldmap.test.ts

```diff
@@ -387,4 +376,46 @@ describe('Worldmap', () => {
     worldMap = new WorldMap(ctrl, document.getElementsByClassName('mapcontainer')[0]);
     worldMap.createMap();
   }
+
+  describe('when apply log is set', () => {
+    const LOW = 1;
+    const HIGH = 3000;
+    const AVG = (HIGH + LOW)/2;
+
+    beforeEach(() => {
+      ctrl.data = new DataBuilder()
+        .withCountryAndValue('SE', LOW)
+        .withCountryAndValue('IE', AVG)
+        .withCountryAndValue('US', HIGH)
+        .withThresholdValues([2])
+        .build();
+      ctrl.panel.circleMinSize = '2';
+      ctrl.panel.circleMaxSize = '10';
+      ctrl.panel.isLogScale = true;
+      ctrl.panel.logFunction = 'log1p';
+      worldMap.drawCircles();
+    });
+
+    it('should draw three circles on the map', () => {
+      expect(worldMap.circles.length).toBe(3);
+    });
+
+    it('should create a circle with min circle size for smallest value size', () => {
+      expect(worldMap.circles[0].options.radius).toBe(2);
+    });
+
+    it('should create a circle with circle size 9 for mid value size', () => {
+      expect(worldMap.circles[1].options.radius).toBe(9);
+    });
+
+    it('should create a circle with max circle size for largest value size', () => {
+      expect(worldMap.circles[2].options.radius).toBe(10);
```

Comment:
Would it make sense to create constants for `MinSize` and `MaxSize` and refer to them in the `beforeEach` and `it` methods?

Reply:
There are tests where those max size and min sizes are different and, even, some tests (I wrote before) that uses their own internal constants. That one was just a copy from a previous test. But I'll create this const anyway ... ;)